### PR TITLE
Fixed multiple styling fixes

### PR DESCRIPTION
**Pull request by harshilsharma63**
LIst of fixes-
1. Handled long board title
2. Handle long board descriptin
3. Disabled horizontal scroil appearing in RHS
4. Fixed verticall scroll not working in RHS
5. Handle RHS text in DM and GM
6. Handled "link" button text for long channel names.

### CODE REVIEW

File: mattermost-plugin/webapp/src/components/rhsChannelBoardItem.scss [hunk 1]
```diff
@@ -22,6 +21,16 @@
...
             flex-grow: 1;
+            white-space: nowrap;
+            overflow: hidden;
+            text-overflow: ellipsis;
         }
     }
+
+    .description {
+        display: -webkit-box;
+        -webkit-line-clamp: 3;
```
Comment (Pinjasaur) [inline mattermost-plugin/webapp/src/components/rhsChannelBoardItem.scss:32]: nice, I don't think I had heard of this property before 🚀 https://caniuse.com/css-line-clamp

**Review by Pinjasaur — APPROVED** [review]
Nice @harshilsharma63! LGTM other than some snapshot tests that need to be updated 🙂

**Comment by asaadmahmood** [comment]
@Pinjasaur @harshilsharma63  I think we can in the future make a mixin for the truncation of a single line, and also multiple lines to provide an easy way to truncate things.

File: mattermost-plugin/webapp/src/components/rhsChannelBoardItem.tsx [hunk 2]
```diff
@@ -68,7 +68,7 @@ const RHSChannelBoardItem = (props: Props) => {
                     </Menu>
                 </MenuWrapper>
             </div>
-            <div>{board.description}</div>
+            <div className='description'>{board.description}</div>
```
[the remark this answers is not in the document]
Comment (harshilsharma63) [reply mattermost-plugin/webapp/src/components/rhsChannelBoardItem.tsx:71]: Hmm, that makes sense. We can follow this in any new component we create.

File: mattermost-plugin/webapp/src/components/rhsChannelBoardItem.scss (hunk 1 above)
Comment (harshilsharma63) [reply mattermost-plugin/webapp/src/components/rhsChannelBoardItem.scss:32]: Yeah, its awesome. Browser support is solid.

**Pull request merged**